components/MultipleChoiceList.js

```diff
@@ -46,26 +45,29 @@ class MultipleChoiceList extends React.Component {
     var template = this.props.template;
 
     return (
-      <ul className={this.props.cssClass}>
+      <div className={this.props.rootClass}>
         {facetValues.map(facetValue => {
           return (
-            <li key={facetValue.name} onClick={this.handleClick.bind(this, facetValue.name)}>
+            <div className={this.props.itemClass} key={facetValue.name} onClick={this.handleClick.bind(this, facetValue.name)}>
               <Template data={facetValue} template={template} />
-            </li>
+            </div>
           );
         })}
-      </ul>
+      </div>
```

Comment:
Yes I should have commented earlier.

So the motivation is that `<ul><li>` have a default CSS most of the time: bullet points.

And in most situations you will want to remove the bullet points for styling. So I thought (and that was the case for my demo) that the easiest was just to use `<div>` everywhere.

But we can rollback to `<ul><li>`